metadata.tex

```diff
@@ -198,19 +192,26 @@
   \end{tikzpicture}
 \end{titlepage}
 
-\title{\includegraphics[width=6cm]{\images/title.png}\\The Rewrite}
+\title{\includegraphics[width=6cm]{\images/title.png}\\
+  \en{The Rewrite}
+  \pl{}
+}
 
 \author{Hermanni "Heegu" Karppela}
 \maketitle
 
 \begin{center}
-  Version 1.0
+  \en{Version 1.0}
```

Comment:
wdyt about moving the version literal to a constant in case there are far more languages?

Reply:
Interesting idea, but how do you imagine this exactly? Sounds a little like you're thinking JavaScript 😆

Reply:
Or, we ditch "Version" and simply do `v1.0`